File: Java/program40.java
import java.util.Scanner;

public class program40 {

    
    public static void Display(int iNo) {
        
        for (int iCnt = 2; iCnt <= iNo; iCnt=iCnt+2) {
            System.out.print(iCnt + "\t");
        }
        System.out.println(); 
    }

    public static void main(String[] args) {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;

        System.out.println("Please enter frequency : ");
        iValue = sobj.nextInt();

        Display(iValue);

        
        sobj.close(); 
    }
}

File: Python/program40.py
def display(i_no):
    # Start at 2, increment by 2
    for i_cnt in range(2, i_no + 1, 2):
        print(f"{i_cnt}\t", end="")
    print()

if __name__ == "__main__":
    try:
        i_value = int(input("Please enter frequency : \n"))
        display(i_value)
    except ValueError:
        print("Invalid input!")